.github/workflows/grass-manual.yml

```diff
@@ -0,0 +1,45 @@
+name: GRASS GIS addon manual
+
+on:
+  workflow_call:
+
+jobs:
+  build-grass-manual:
+    name: build-grass-manual
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Creation of GRASS GIS addon manual
+        run: |
+          ADDON_NAME=$(echo ${GITHUB_REPOSITORY} | cut -d "/" -f 2)
+          wget -O manual_Dockerfile https://raw.githubusercontent.com/mundialis/github-workflows/main/manual-docker/Dockerfile
+          docker build -f manual_Dockerfile --build-arg ADDON_NAME=$ADDON_NAME --build-arg GITHUB_REPOSITORY=$GITHUB_REPOSITORY -t grass-addon-manual .
+          ID=$(docker create grass-addon-manual)
+          echo $ID
+          docker cp $ID:/src/build/docs/html/ public
+      - name: Upload Pages artifact
+        uses: actions/upload-pages-artifact@v3
+        with:
+          path: "public"
+
+  deploy-grass-manual:
+    name: deploy-grass-manual
+    runs-on: ubuntu-latest
+    needs: build-grass-manual
+
+    permissions:
+      pages: write
+      id-token: write
+
+    environment:
+      # environment created automatically by GitHub
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
```

Comment:
Are the variables `name` and `url` used anywhere?

Reply:
I think `actions/deploy-pages@v4` uses it.